agents/available-connections.mdx

```diff
@@ -161,7 +161,17 @@ agent card for the full list.
       />
       <span className="text-lg font-semibold">Supabase</span>
     </div>
-    Supabase is an open‑source Firebase alternative.
+    Supabase is the PostgresSQL development platform.
+
+    <p className="mt-4">
+      <a
+        className="underline text-pink-600 font-bold"
+        href="/agents/connections/supabase"
+      >
+       Using Supabase with Hypermode 👉
```

Comment:
Can we make this a button that just says "Details"?

Also, maybe we add a little pill with the number of tools for this connection?

Like "7 tools"?